analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -176,6 +176,12 @@
     </ProjectReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Common\Resources\SonarLint.xml">
```

Comment:
I would say this can be in the same itemgroup with <None Include="Common\Resources\input.txt"> and maybe they can be merged with some Common\Resources\*  pattern.  Besides that I am not sure an always copy is needed. maybe preserveNewest would be good enough.

Reply:
preserveNewest is good enough indeed. I also moved it near `Common\Resources\input.txt`